role-odcs.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Setup ODCS frontend
```

Comment:
can we have role-odcs-frontend.yml and role-odcs-backend.yml, themselves calling corresponding role ? Just to keep the logic we use so far, as automated ansible deploy will try to ensure each git repo/role is up2date before kicking ansible-playbook for the role